whipper/command/cd.py

```diff
@@ -455,7 +461,16 @@ def _ripIfNotRipped(number):
                                     self.ittoc.getTrackLength(number), number)
 
             self.program.saveRipResult()
-
+        try:
+            excluded_tracks = self.options.excluded_tracks.split(",")
+        except UnboundLocalError:
+            excluded_tracks = []
+        excluded_tracks_integer = []
+        for i in excluded_tracks:
+            try:
+                excluded_tracks_integer.append(int(i.replace(" ", "")))
+            except ValueError:
+                pass
```

Comment:
Instead of silently ignoring errors, we should abort at argument-handling time with a helpful yet short error message.